### PR TITLE
Create array-reverse() doc, closes #3162

### LINKED ISSUE #3162: [Term Entry] PHP Arrays array_reverse()
### Reason/inspiration (optional)

We would like a new term entry for [PHP](https://www.codecademy.com/resources/docs/php) about the `array_reverse()` function.

The entry should go in a new file in `docs/content/php/concepts/arrays/terms/array-reverse/array-reverse.md` (Note the dashes rather than underscores) 

The entry should 

- Have a syntax section illustrating the function with a ` ```pseudo ` code block. 
- An example showing the function in use with a ` ```php ` code block. 
- A runnable codebyte example in a ` ```codebyte/php` block. 

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by reganmarie**
### Description

- Created array_reverse.md on filepath docs/content/php/concepts/arrays/terms/array-reverse/array-reverse.md
- Added Title, description Subjects, Tags, and Catalog Content
- Added description, syntax, example, example using optional argument, and codebyte example
- Ran through 3 online plagiarism checkers to ensure 100% unique wording and description, passed with 100%
- Followed content and style guidelines
- Checked on Google docs with _ and `` removed to ensure accurate spellings of all words

### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/php/concepts/arrays/terms/array-reverse/array-reverse.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+## Syntax
+
+```pseudo
+array_reverse(array, preserve_keys)
+```
+
+- `array`: The array to reverse.
+- `preserve_keys`: Optional argument that defaults to `false` if omitted. If set to `true`, the keys of the original, non-reversed array will be preserved.
+> **Note:** If the keys used are non-numeric, they will always be preserved, regardless of whether the `preserve_keys` argument is used.
```
Comment (caupolicandiaz) [inline content/php/concepts/arrays/terms/array-reverse/array-reverse.md:26]: add an empty line between the last parameter and the note

File: content/php/concepts/arrays/terms/array-reverse/array-reverse.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+## Example
+
+This example returns the reversed array `$a` using the `array_reverse()` method. In this example, the `preserve_keys` argument is not used.
```
Comment (caupolicandiaz) [inline content/php/concepts/arrays/terms/array-reverse/array-reverse.md:30]:
```suggestion
This example returns the reversed array `$a` using the `array_reverse()` method. 
```

File: content/php/concepts/arrays/terms/array-reverse/array-reverse.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+```php
+<?php
+  $a = array(0=>"andrew",1=>"byrne",2=>"hozier",3=>"unreal");
+  echo $backward = array_reverse($a);
```
Comment (caupolicandiaz) [inline content/php/concepts/arrays/terms/array-reverse/array-reverse.md:35]:
```suggestion
  print_r(array_reverse($a));
```

File: content/php/concepts/arrays/terms/array-reverse/array-reverse.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+  [3] => andrew
+)
+```
+
+This example uses the `preserve_keys` argument while calling the `array_reverse()` method to return `$a`.
+
+```php
+<?php
+    $a = array(0=>"andrew",1=>"byrne",2=>"hozier",3=>"unreal");
+    echo $backward = array_reverse($a, true);
```
Comment (caupolicandiaz) [inline content/php/concepts/arrays/terms/array-reverse/array-reverse.md:56]:
```suggestion
    print_r(array_reverse($a, true));
```

File: content/php/concepts/arrays/terms/array-reverse/array-reverse.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+Array
+(
+  [3] => unreal
+  [2] => hozier
+  [1] => byrne
+  [0] => andrew
+)
+```
+
+## Codebyte Example
```
Comment (caupolicandiaz) [inline content/php/concepts/arrays/terms/array-reverse/array-reverse.md:72]: The codebyte is a duplicated of the first example. I would suggest deleting it or updating the example to demonstrate some new functionality, use-case, etc

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@reganmarie I've completed my initial review. Please update the entry, and tag me when you've finished making changes. Thanks

File: content/php/concepts/arrays/terms/array-reverse/array-reverse.md (hunk 1 above)
Comment (reganmarie) [reply content/php/concepts/arrays/terms/array-reverse/array-reverse.md:26]: Done!

File: content/php/concepts/arrays/terms/array-reverse/array-reverse.md (hunk 3 above)
Comment (reganmarie) [reply content/php/concepts/arrays/terms/array-reverse/array-reverse.md:35]: Fixed!

File: content/php/concepts/arrays/terms/array-reverse/array-reverse.md (hunk 4 above)
Comment (reganmarie) [reply content/php/concepts/arrays/terms/array-reverse/array-reverse.md:56]: Fixed!

File: content/php/concepts/arrays/terms/array-reverse/array-reverse.md (hunk 5 above)
Comment (reganmarie) [reply content/php/concepts/arrays/terms/array-reverse/array-reverse.md:72]: Updated!

**Comment by reganmarie** [comment]
@caupolicandiaz Revised and committed here: [Updated based on review in PR](https://github.com/Codecademy/docs/commit/9446960222efc8328e10c20e7cb2f9156666736b)
Please let me know if there's anything else I can do or provide for your next review!

**Review by caupolicandiaz — APPROVED** [review]
@reganmarie I just made a couple of edits to the codebyte. This is ready for a second review.

**Comment by reganmarie** [comment]
Thank you @caupolicandiaz ! Please let me know if there's anything I can do in the meantime :)

**Review by SSwiniarski — APPROVED** [review]
@reganmarie Looks good.

**Pull request merged**